src/bindings/NodeOs.re

```diff
@@ -0,0 +1,2 @@
+[@module "node:os"]
+external availableParallelism: unit => int = "availableParallelism";
```

Comment:
Do I understand it right- `availableParallelism` value could be used to set `chunkSize`?

Reply:
@ixzzd 
Right.
It can be used to split the initial pages array to chunks. Then we take the chunk for example with 8 pages and run 8 workers in parallel to build pages. After workers finished the build of the first chunk, we build next chunk, etc.
But `chunkSize` value should be reasonably limited because memory consumption will be huge if we try to run something like 64 workers simultaneously :). Node will fail.